core/src/main/scala/precog/SbtPrecogBase.scala

```diff
@@ -143,7 +143,7 @@ abstract class SbtPrecogBase extends AutoPlugin {
       autoCompilerPlugins := true,
       autoAPIMappings := true,
 
-      addCompilerPlugin("org.typelevel" %% "kind-projector"     % "0.13.0" cross CrossVersion.full),
+      addCompilerPlugin("org.typelevel" %% "kind-projector"     % "0.13.2" cross CrossVersion.full),
```

Comment:
Do we want 0.13.2 or 0.13.1? I don't see 0.13.2 in [Maven](https://repo1.maven.org/maven2/org/typelevel/kind-projector_2.13.6/).

Reply:
Ah I saw 0.13.2 in the releases notes on the repo (with a note saying it's identical to 0.13.1) so I'll switch to 0.13.1 given that it wasn't actually released to maven central.